rma/models/stock_warehouse.py

```diff
@@ -30,6 +30,7 @@ class StockWarehouse(models.Model):
     )
     rma_in_route_id = fields.Many2one("stock.route", "RMA in Route")
     rma_out_route_id = fields.Many2one("stock.route", "RMA out Route")
+    rma_out_replace_route_id = fields.Many2one("stock.route", "RMA out Replace Route")
```

Comment:
@sbejaoui Should'nt we use the mechanisms to create that route (see `_get_route_values()`) ?

Reply:
I believe the common use case is to use the default delivery route, which is why I didn't create or assign a custom route to the warehouse. If more people feel that having a custom route for RMA delivery by default is necessary, I will make the change.